src/notifications/builder.ts

```diff
@@ -168,8 +174,11 @@ export class NotificationBuilder {
   }
 
   attachmentPost(post: Reference<Post>): NotificationBuilder {
+    const type =
+      postTypeToAttachmentType[post.type] ?? NotificationAttachmentType.Post;
```

Comment:
We might sync this after the notification creation with trigger because shared post do not get video type immediately.

Reply:
Added here 9c3fd2642e5e5ce17b629df8a88b15b58353ecb0